test-settings.example.json

```diff
@@ -1,20 +1,4 @@
 {
```

Comment:
Why did you remove this file in this PR?

Reply:
The file is still there, I just removed settings for HTTP and SSH connections.

Reply:
Yes, you are correct. My bad!